infra/nixos/concrexit.nix

```
@@ -312,6 +312,26 @@ in
                 alias = "${cfg.dir}/media/";
                 extraConfig = "internal;";
               };
+              locations."/apple-app-site-association" = {
```

```suggestion
              locations."/.well-known/apple-app-site-association" = {
```
Although having the file in the root path probably works, it's preferred to have it at `.well-known` (this is also checked first).